LT.DigitalOffice.WikiService/.dockerignore

```diff
@@ -0,0 +1,2 @@
+bin/
```

Comment:
docker ignore file in other services is different, make like those service or approve this file with Mariya

Reply:
It's strange. It was done like in some other service. But now can't find. OK, I'll change it.